packages/libllvm/libcompiler-rt.subpackage.sh

```diff
@@ -6,7 +6,7 @@ lib/clang/*/include/fuzzer/FuzzedDataProvider.h
 lib/clang/*/include/profile/InstrProfData.inc
 lib/clang/*/include/sanitizer/
 lib/clang/*/include/xray/
-lib/clang/*/lib/android/
+lib/clang/*/lib/linux/
```

Comment:
Compiler-rt moved where it places its libraries by default, llvm/llvm-project@a68ccba, so change this and the Driver change above is no longer needed.